client/src/pages/Registration/RegistrationFields.jsx

```diff
@@ -5,51 +5,22 @@
 // errorMessage: The message to show if the field is missing but is required
 // onChanged(value, disableFields): disable/enable fields based on a value (dependancies)
 // className: the class name applied around the child form input component
-
 export const fields = {
   General: {
-    // nameLabel: {
-    //   type: 'label',
-    //   label: "Full Name? (First & Last)",
-    //   isRequiredInput: true,
-    // },
-    // firstName: {
-    //   type: 'text',
-    //   inputType: 'text',
-    //   placeholder: 'John',
-    //   hasRestrictedInput: true,
-    //   isRequiredInput: true,
-    //   errorMessage: 'Please enter a valid name',
-    //   localStorageKey: 'registration-firstName',
-    //   className: "half-width-input",
-    // },
-    // lastName: {
-    //   type: 'text',
-    //   inputType: 'text',
-    //   placeholder: 'Doe',
-    //   hasRestrictedInput: true,
-    //   isRequiredInput: true,
-    //   errorMessage: 'Please enter a valid name',
-    //   localStorageKey: 'registration-lastName',
-    //   className: "half-width-input",
-    // },
-    // preferredName: {
-    //   type: 'text',
-    //   inputType: 'text',
-    //   label: 'preferred name?',
-    //   placeholder: 'Joey',
-    //   hasRestrictedInput: true,
-    //   isRequiredInput: false,
-    //   errorMessage: '',
-    //   localStorageKey: 'registration-preferredName',
-    //   className: "half-width-input",
-    // },
+    name: {
+      type: 'text',
+      inputType: 'text',
+      placeholder: 'John Doe',
+      label: 'Legal Name',
+      isRequiredInput: true,
+      errorMessage: 'Please enter your legal name',
+    },
     pronoun: {
       type: 'dropdown',
       label: 'Pronoun',
-      values: ['he/him', 'she/her', 'they/them', 'other'],
+      values: ['Prefer Not to Say', 'he/him', 'she/her', 'they/them', 'Other'],
```

Comment:
Whoever is working on #131 will need to update the options in the `FroshGroupSchema` to reflect these pronouns, otherwise the algorithm will break